src/py42/services/alerts.py

```diff
@@ -31,21 +31,18 @@ def get_details(self, alert_ids):
         results = self._connection.post(uri, json=data)
         return _convert_observation_json_strings_to_objects(results)
 
-    def resolve(self, alert_ids, reason=None):
+    def update_state(self, state, alert_ids, reason=None):
```

Comment:
Since this is a new method, can we have the parameter name be `note` instead of `reason`? Now that there are more states, `note` makes more sense.

Reply:
It is ok for `resolve` and `reopen` to use `reason` but the `update_state` methods should use `note`.

Reply:
And pass `reason` as note in `resolve` and `reopen`

Reply:
renamed.